dashboard/app/models/level.rb

```diff
@@ -59,8 +62,13 @@ def self.model_name
 
   # https://github.com/rails/rails/issues/3508#issuecomment-29858772
   # Include type in serialization.
+  # also include level_concept_difficulty
```

Comment:
Can you explain why the addition of `include => :level_concept_difficulty` was needed here, if it's not related to the link in the comment? I'm inclined to try and find another way of dealing with whatever the issue was here further up the call-stack, since this is a hack applied on top of a hack which can get hard to maintain/debug down the road.